README.md

```diff
@@ -24,6 +24,48 @@ Then build the `pace` docker image at the top level.
 make build
 ```
 
+## Building Bare-Metal
+
+Pace requires gcc > 9.2, cuda, MPI, and Python 3.8 on your system in order to run bare-metal.
+
+You will also need the headers of the boost libraries in your `path` (boost itself does not need to be installed).
+
+```shell
+cd $BOOST_LOCATION
+wget https://boostorg.jfrog.io/artifactory/main/release/1.79.0/source/boost_1_79_0.tar.gz
+tar -xzf boost_1_79_0.tar.gz
+mkdir -p boost_1_79_0/include
+mv boost_1_79_0/boost boost_1_79_0/include/.
+setenv BOOST_ROOT $BOOST_LOCATION/boost_1_79_0
+```
+
+After cloning Pace you will need to update the repository's submodules:
+```shell
+$ git submodule update --init
+```
+
+We recommend creating a python venv or conda environment specifically for Pace.
```

Comment:
Can you include example instructions for this, using `python3 -m venv venv` (since this is built-in)?